terraform/main.tf

```diff
@@ -54,7 +59,12 @@ module "cloud_run" {
   ingress               = var.enable_gclb ? "internal-and-cloud-load-balancing" : "all"
   secrets               = ["github-application-id", "github-installation-id", "github-privatekey"]
   service_account_email = google_service_account.run_service_account.email
-  service_iam           = var.service_iam
+  service_iam = {
+    admins     = var.service_iam.admins
+    developers = toset(flatten([var.ci_service_account_member, var.service_iam.developers]))
```

Comment:
I believe this is missing brackets, but also this reads funny to me, instead I would concatenate the values:

```terraform
    developers = toset(concat(var.service_iam.developers, [var.ci_service_account_member]))
```